packages/widget-chat/src/components/list-separator/styles.css

```diff
@@ -0,0 +1,60 @@
+:root {
+  --color-white: #fefefe;
+  --color-primary: #049fd9;
+  --color-gray-light: #aeaeaf;
+  --color-gray: #858688;
+  --color-gray-ultra-light: #d7d7d8;
+}
```

Comment:
👍  Just grabbed it from another css file. Wonder if there is an unused-vars option in style lint?